db/migrate/20180904120211_create_payloads.rb

```diff
@@ -0,0 +1,19 @@
+class CreatePayloads < ActiveRecord::Migration
+  def change
+    create_table :payloads do |t|
+      t.string :name
+      t.string :uuid
+      t.integer :timestamp
+      t.string :arch
+      t.string :platform
+      t.string :urls
+      t.string :description
+      t.references :workspace
```

Comment:
I didn't see this being done in any of the `db/migrate/` files and a number of the models `belongs_to :workspace`. Did I overlooked something? Why is this need here an not in other models?

Reply:
It's just an abstracted way of including the `workspace_id` column. Other (older) migration files just explicitly add the `workspace_id` column. Using `t.references` makes us more future proof in case something is enhanced here in rails.